### ISSUE TITLE
Problem with how themes are listed on config page

### ISSUE THREAD

**Issue by stpaultim** [backdrop-contrib/stickynav#14]
The list of themes is breaking up into two groups. Not sure why. 

![image](https://user-images.githubusercontent.com/3144571/149604054-6eb4343f-cb00-494c-a4a6-a6d3e265d2a5.png)

**Comment by yorkshire-pudding** [comment]
@stpaultim - I think this is because you get settings for each theme. If you now tick Pelerine, then the settings should appear between the Pelerine tick box and the Seven tick box.

**Comment by laryn** [comment]
@stpaultim I agree it's not the most intuitive UI but the summary by @yorkshire-pudding is correct, I believe.
